webapp/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -157,21 +147,17 @@ def show_bug_link(bug_id):
     return markupsafe.Markup(tmpl) % data
 
 
-EXTRA_NEWLINES_RE = re.compile(r"\n\n\n+")
-
 MAX_TITLE_LENGTH = 255
 
 
 @library.global_function
-def generate_create_bug_url(
-    request, template, raw_crash, report, parsed_dump, crashing_thread
-):
-    parsed_dump = parsed_dump or {}
-
+def generate_create_bug_url(crash_report_url, template, processed_crash):
```

Comment:
It's not wildly clear but a "jinja function" is a Python function that's wrapped with the decorator that registers the function with the Jinja engine so it can be used in templates.